Aspire Budgeting/Facilities/AppCoordinator.swift

```diff
@@ -0,0 +1,69 @@
+//
+// AppCoordinator.swift
+// Aspire Budgeting
+//
+
+import Foundation
+import Combine
+import SwiftUI
+
+final class AppCoordinator: ObservableObject {
+  private let objectCreator: ObjectCreator
+  private let stateManager: AppStateManager
+  private let localAuthManager: LocalAuthorizationManager
+  private let userManager: AppUserManager
+  private var stateManagerSink: AnyCancellable!
+  
+  public var isLoggedOut: Bool {
+    stateManager.isLoggedOut
+  }
+  
+  public var needsLocalAuth: Bool {
+    stateManager.isLocalAuthNeeded
+  }
+
+//  @Published private(set) var currentState = AppState.loggedOut
+
+  init(objectCreator: ObjectCreator) {
+    self.objectCreator = objectCreator
+    self.stateManager = objectCreator.stateManager
+    self.localAuthManager = objectCreator.localAuthorizationManager
+    self.userManager = objectCreator.userManager
+  }
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/d909f41c73c54120b2fbb4e96bb79560130cb154/Aspire Budgeting/Facilities/AppCoordinator.swift#L33"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>AppCoordinator.swift:33</code></td>
    </tr>
  </tbody>
</table>